apps/web/src/app/raidpool/_component/loot-item.tsx

```
@@ -0,0 +1,117 @@
+  rarity: string
+  subtype: string
+  icon: { format: string; value: string } | null
+  region?: string
+}
+
+function getItemIcon(itemType: string, subtype?: string) {
+  switch (itemType) {
+    case "GearItem":
+      return <Sword className="h-5 w-5" />
+    case "AspectItem":
+      const className = subtype?.split('Aspect')[0].toLowerCase()
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)